src/utils/helpers.ts

```diff
@@ -29,31 +28,18 @@ export function checkNodeLinkable(
 
 export interface CoordProps {
   zoom: number;
-  scrollXY: [number, number];
-  layout: ElkRoot;
-  containerWidth: number;
-  containerHeight: number;
+  layoutXY: [number, number];
   containerRef: RefObject<HTMLDivElement | null>;
 }
 
 /**
  * Given various dimensions and positions, create a matrix
  * used for determining position.
  */
-export function getCoords({
-  zoom,
-  scrollXY,
-  layout,
-  containerWidth,
-  containerHeight,
-  containerRef
-}: CoordProps) {
+export function getCoords({ zoom, layoutXY, containerRef }: CoordProps) {
   const { top, left } = containerRef.current.getBoundingClientRect();
-  const offsetX = scrollXY[0] - containerRef.current.scrollLeft;
```

Comment:
Can you provide some description to why this was removed? Seems like it was here for a reason ( though I forget now ).